data/examples/declaration/value/function/infix/esqueleto-0-out.hs

```diff
@@ -0,0 +1,12 @@
+import Database.Esqueleto.Experimental
+
+foo = select $ do
+  t <-
+    from $
+      table @Bar
+        `innerJoin` table @Baz
+          `on` do
```

Comment:
Shouldn't these be indented at the same level? Is this an example of the Hoogle module reexport deficiency?

Reply:
Actually I think Ormolu is doing the right thing here given that `innerJoin` is `infixl 2` and `on` is `infix 9` :thinking: The other Esqueleto example imports from a non-existant module `Database.Persist.Esqueleto` (taken from an issue) and that's why it formats differently. I think I will change both to import from `Database.Esqueleto.Experimental`.